plugins/modules/ndo_node_setting.py

```diff
@@ -0,0 +1,391 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Sabari Jaganathan (@sajagana) <sajagana@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: ndo_node_setting
+short_description: Manage Fabric Policies Node Settings on Cisco Nexus Dashboard Orchestrator (NDO).
```

Comment:
```suggestion
short_description: Manage Fabric Policy Node Settings on Cisco Nexus Dashboard Orchestrator (NDO).
```

Should this be Fabric Policy Node Settings everywhere instead of "Policies"?